de-DE/brain-game/Brain Game - notes.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Gehirntraining — Volunteer Notes
```

Comment:
I am going to go ahead and merge this PR but we do have some character issues like so @Matella15